client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-list-renderer.tsx

```diff
@@ -28,24 +27,12 @@ interface PatternListRendererProps {
 const PLACEHOLDER_HEIGHT = 100;
 const MAX_HEIGHT_FOR_100VH = 500;
 
-const PatternListItem = ( {
-	pattern,
-	className,
-	isFirst,
-	isShown,
-	onSelect,
-}: PatternListItemProps ) => {
+const PatternListItem = ( { pattern, className, isShown, onSelect }: PatternListItemProps ) => {
 	const [ inViewOnce, setInViewOnce ] = useState( false );
 	const ref = useInView< HTMLButtonElement >( () => setInViewOnce( true ), {
 		threshold: [ 0 ],
 	} );
 
-	useEffect( () => {
-		if ( isShown && inViewOnce && isFirst && ref.current ) {
```

Comment:
I guess we can simply remove `inViewOnce` from the condition to fix the issue. WDYT?

Reply:
OKay 👍